packages/common/src/ui-common/index.ts

```diff
@@ -9,6 +10,15 @@ export {
     EnvironmentMode,
     DependencyName,
 } from "./environment";
+export {
+    AbstractHttpClient,
+    FetchHttpClient,
+    HttpClient,
+    HttpRequestInit,
+    MockHttpClient,
```

Comment:
IMO let's not export mocks/fakes in index.ts. I *think* this still causes issues where webpack can't treeshake out these exports.

Reply:
Makes sense. Should I do the same with `getMockEnvironment` and `initMockEnvironment` at [the top of the file](https://github.com/Azure/BatchExplorer/blob/vnext/packages/common/src/ui-common/index.ts#L4)?